specification/billing/resource-manager/readme.md

```diff
@@ -4,41 +4,267 @@
 
 > `autorest`
 
 To see additional help and options, run:
 
 > `autorest --help`
+
 ---
 
 ## Configuration
 
-
-
 ### Basic Information
+
 These are the global settings for the Billing API.
 
-``` yaml
+```yaml
 openapi-type: arm
-tag: package-2020-05
+tag: package-2024-04
+```
+
+### Tag: package-2024-04
+
+These settings apply only when `--tag=package-2024-04` is specified on the command line.
+
+These settings apply only when `--tag=package-2024-04` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-04'
+title: BillingManagementClient
+description: Billing Client
+input-file:
+- Microsoft.Billing/stable/2024-04-01/agreement.json
+- Microsoft.Billing/stable/2024-04-01/associatedTenant.json
+- Microsoft.Billing/stable/2024-04-01/availableBalance.json
+- Microsoft.Billing/stable/2024-04-01/billingAccount.json
+- Microsoft.Billing/stable/2024-04-01/billingPermission.json
+- Microsoft.Billing/stable/2024-04-01/billingProfile.json
+- Microsoft.Billing/stable/2024-04-01/billingProperty.json
+- Microsoft.Billing/stable/2024-04-01/billingRequest.json
+- Microsoft.Billing/stable/2024-04-01/billingRoleAssignment.json
+- Microsoft.Billing/stable/2024-04-01/billingRoleDefinition.json
+- Microsoft.Billing/stable/2024-04-01/billingSavingsPlan.json
+- Microsoft.Billing/stable/2024-04-01/billingSubscription.json
+- Microsoft.Billing/stable/2024-04-01/customer.json
+- Microsoft.Billing/stable/2024-04-01/department.json
+- Microsoft.Billing/stable/2024-04-01/enrollmentAccount.json
+- Microsoft.Billing/stable/2024-04-01/invoice.json
+- Microsoft.Billing/stable/2024-04-01/invoiceSection.json
+- Microsoft.Billing/stable/2024-04-01/operation.json
+- Microsoft.Billing/stable/2024-04-01/payment.json
+- Microsoft.Billing/stable/2024-04-01/policy.json
+- Microsoft.Billing/stable/2024-04-01/product.json
+- Microsoft.Billing/stable/2024-04-01/reservation.json
+- Microsoft.Billing/stable/2024-04-01/transaction.json
+- Microsoft.Billing/stable/2024-04-01/transfers.json
+- Microsoft.Billing/stable/2024-04-01/types.json
+suppressions:
+- code: AllProxyResourcesShouldHaveDelete
+  from: billingRequest.json
+  reason: Service design forces behavior
+- code: AllProxyResourcesShouldHaveDelete
+  from: billingSubscription.json
+  reason: Service design forces behavior
+- code: AllProxyResourcesShouldHaveDelete
+  from: policy.json
+  reason: Service design forces behavior
+- code: AllProxyResourcesShouldHaveDelete
+  from: transfers.json
+  reason: Service design forces behavior
+- code: AllTrackedResourcesMustHaveDelete
+  from: reservation.json
+  reason: Breaking change
+- code: AvoidAdditionalProperties
+  from: billingRequest.json
+  reason: Service design forces behavior
+- code: AvoidAdditionalProperties
+  from: billingSavingsPlan.json
+  reason: False positive. Used for "tags"
+- code: AvoidAdditionalProperties
+  from: billingSubscription.json
+  reason: Breaking change
+- code: AvoidAdditionalProperties
```

Comment:
You should also remove the suppressions for files you do not have to.

Reply:
Missed out removed the suppressions which are not needed